avocado/core/test.py

```diff
@@ -72,7 +72,7 @@ def __init__(self, uid, name, variant=None, no_digits=None):
             self.str_uid = str(uid).zfill(no_digits if no_digits else 3)
         else:
             self.str_uid = str(uid)
-        self.name = name or "<unknown>"
+        self.name = name
```

Comment:
Good point.  I think I had the intentions to do something extra here (or not).  Let me think about it again.